app/models/coronavirus/sub_section/structured_content.rb

```diff
@@ -0,0 +1,73 @@
+class Coronavirus::SubSection::StructuredContent
+  HEADER_PATTERN = PatternMaker.call(
+    "starts_with hashes then perhaps_spaces then capture(text) and nothing_else",
+    hashes: "#+",
+    text: '\w.+',
```

Comment:
The title field actually comes from [`groups`](https://github.com/alphagov/collections-publisher/blob/dc4153154bc4fe05b0e997552e628e86500e597f/app/models/coronavirus/sub_section/structured_content.rb#L60) rather than here so no change needed :)

We'll take a look at the `PatternMaker` stuff later today.

Reply:
I've changed the parse tests to check for specific output after a conversation on slack about the complexity that faffing about with PatternMaker requires 😂